TKGame/Enemy.cs

```diff
@@ -63,18 +65,50 @@ public override void Update(GameTime gameTime)
 
         public void Move()
         {
-            Position.X += Velocity.X;                       // Enemy initially starts moving to right
+            // getter for player class
+            Player player = EntityManager.GetEntities().FirstOrDefault(x => x.entityName == "player" && x is Player) as Player;
 
-            if (Position.X == rightBoundary)
+            // check if player is null
+            if (player != null)
             {
-                Velocity.X = -speed.X;                      // Enemy moves left
-                Orientation = SpriteEffects.FlipHorizontally;
-            }
-            else if (Position.X == leftBoundary)
-            {
-                Velocity.X = speed.X;                       // Enemy moves right
-                Orientation = SpriteEffects.None;
+                Vector2 playerPosition = player.Position;
+                
+                if (Position.X > playerPosition.X)
+                {
+                    Position.X -= speed.X;
+                    Orientation = SpriteEffects.FlipHorizontally;
+                }
+                    
+                if (Position.X < playerPosition.X)
+                {
+                    Position.X += speed.X;
+                    Orientation = SpriteEffects.None;
+                }
```

Comment:
an else if could be better, not sure on the efficiency difference here but this stuck out as strange when I first read it. Other than that this is fine, refactoring in the future definitely possible